<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>得意黑</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>服务器地址:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>得意黑</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>得意黑</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="PrimaryPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>153</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="LineEdit" name="serverEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.158.128</string>
   </property>
  </widget>
  <widget class="LineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="LineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
